packages/local-cli/package.json

```diff
@@ -27,10 +27,10 @@
   },
   "dependencies": {
     "@babel/core": "^7.1.5",
+    "@babel/plugin-transform-strict-mode": "^7.2.0",
```

Comment:
oh, forgot we use babel/register here. time to make a proper build/watch infra :p

Reply:
Yeah, so called "post-merge" review :joy: